macro/inspectGeant4Geo.py

```
@@ -22,7 +22,11 @@
 modules = shipDet_conf.configure(run, ShipGeo)
 run.SetUserConfig("g4Config.C")
 run.SetName("TGeant4")
-run.SetSink(ROOT.FairRootFileSink(ROOT.TMemFile("output", "recreate")))
+import tempfile
+
+sink = ROOT.FairRootFileSink(tempfile.mktemp(suffix=".root"))
```

Ruff (0.15.13)

[error] Use of insecure and deprecated function (`mktemp`)

(S306)